test/integration/shoots/logging/logging.go

```diff
@@ -0,0 +1,266 @@
+	gardenNamespace := &corev1.Namespace{}
+	fluentBit := &appsv1.DaemonSet{}
+	fluentBitConfMap := &corev1.ConfigMap{}
+	fluentBitService := &corev1.Service{}
+	fluentBitClusterRole := &rbacv1.ClusterRole{}
+	fluentBitClusterRoleBinding := &rbacv1.ClusterRoleBinding{}
+	fluentBitServiceAccount := &corev1.ServiceAccount{}
+	clusterCRD := &apiextensionsv1.CustomResourceDefinition{}
+	lokiSts := &appsv1.StatefulSet{}
+	lokiServiceAccount := &corev1.ServiceAccount{}
+	lokiService := &corev1.Service{}
+	lokiConfMap := &corev1.ConfigMap{}
+
+	framework.CBeforeEach(func(ctx context.Context) {
+		checkRequiredResources(ctx, f.SeedClient)
+		//Get the DaemonSet
+		err := f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitName}, fluentBit)
+		framework.ExpectNoError(err)
+		//Get the ConfigMap
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitConfigMapName}, fluentBitConfMap)
+		framework.ExpectNoError(err)
+		//Get Service
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitName}, fluentBitService)
+		framework.ExpectNoError(err)
+		//Get ClusterRole
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitClusterRoleName}, fluentBitClusterRole)
+		framework.ExpectNoError(err)
+		//Get Rolebinding
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitClusterRoleName}, fluentBitClusterRoleBinding)
+		framework.ExpectNoError(err)
+		//Get ServiceAccount
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitName}, fluentBitServiceAccount)
+		framework.ExpectNoError(err)
+		//Get cluster CRD
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: "", Name: "clusters.extensions.gardener.cloud"}, clusterCRD)
+		framework.ExpectNoError(err)
+		//Get Loki StS
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiName}, lokiSts)
+		framework.ExpectNoError(err)
+		//Get Loki ServiceAccount
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiName}, lokiServiceAccount)
+		framework.ExpectNoError(err)
+		//Get Loki Service
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiName}, lokiService)
+		framework.ExpectNoError(err)
+		//Get Loki Service
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiConfigMapName}, lokiConfMap)
+		framework.ExpectNoError(err)
+	}, initializationTimeout)
+
+	f.Beta().CIt("should get container logs from loki for all namespaces", func(ctx context.Context) {
```

Comment:
it deploys a lot of resources which may influence other tests.
Therefore, I would suggest to execute in serial.
```suggestion
	f.Beta().Serial().CIt("should get container logs from loki for all namespaces", func(ctx context.Context) {
```